assets/images/active-cell.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Where did these SVGs come from? I'm guessing ionicons but just wanting to double check.

Reply:
I got these from figma designs